Source/Diagnostics/WarpXIO.cpp

```diff
@@ -746,29 +746,72 @@ WarpX::WriteJobInfo (const std::string& dir) const
 
 
 /* \brief
- *  The slice is ouput using visMF and can be visualized used amrvis. 
+ *  The raw slice data is written out in the plotfile format and can be visualized using yt. *  The slice data is written to diags/slice_plotfiles/pltXXXXX at the plotting interval.  
```

Comment:
Looks like there's a line break missing here.

Reply:
Thank you for catching that. Fixed it now.

Reply:
Thank you Maxence for reviewing this PR. This branch has only been tested without Mesh Refinement. The slice generation currently works with one level only and therefore has not been tested with Mesh refinement. I have tested the slice generation for 1D 2D and 3D slices and visualized it using amrvis. I have tested visualization with 3D using yt. I will update here after I test yt-visualization with 2D and 1D configurations. I dont think I have changed the default input parameters for sub-sampling.